travis/run_onnx_model_artifact_tests.sh

```diff
@@ -11,7 +11,7 @@ cd $GIT_ROOT
 
 # Install required packages for onnx model artifacts test
 pip install onnx onnxruntime skl2onnx
-pytest $GIT_ROOT/tests/artifact/test_onnx_model_artifact.py
+pytest $GIT_ROOT/tests/artifact/test_onnx_model_artifact.py --cov=bentoml
```

Comment:
what does --cov=bentoml do here? 

Does it count this test's coverage to the overall coverage score?

Reply:
Yes. I think the codecov bot already update the message in this PR.  Check above we have artifact coverage now!

Reply:
nice!